tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -370,12 +385,16 @@ def update_targets(self, data: bytes) -> None:
         Raises:
             RepositoryError: Metadata failed to load or verify. The actual
                 error type and content will contain more details.
+
+        Returns:
+            Deserialized and verified targets Metadata object
         """
-        self.update_delegated_targets(data, "targets", "root")
+        targets = self.update_delegated_targets(data, "targets", "root")
+        return targets
```

Comment:
trivial nitpick: unnecessary variable

Reply:
I wondered here and decided to add it for better readability, but I'm fine with removing it.